Lib/test/test_plistlib.py

```diff
@@ -226,6 +226,7 @@ def test_indentation_dict_mix(self):
     def test_uid(self):
         data = UID(1)
         self.assertEqual(plistlib.loads(plistlib.dumps(data, fmt=plistlib.FMT_BINARY)), data)
+        self.assertEqual(plistlib.loads(plistlib.dumps(data, fmt=plistlib.FMT_XML)), data)
```

Comment:
Using `for fmt in ALL_FORMATS:` here would better match the style of the surrounding tests